.github/workflows/ci_workflows.yml

```diff
@@ -93,6 +93,10 @@ jobs:
 
         - name: Python 3.11 Double test (Run tests twice)
```

Comment:
```suggestion
        - name: Python 3.11 Double test (Run tests twice) [aarch64]
```

Reply:
But then I have to go update the branch protection and all the PRs will need rebasing. 😬

Reply:
@bsipocz and @mhvk , any preference about keeping or changing job name?

Reply:
Happy to leave as is to avoid giving people pain, but even happier to leave the decision up to you!

Reply:
Maybe we can do it when we bump Python version in the future since the name would change anyway...